specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2025-03-01/examples/VolumeGroups_PreBackup_MaximumSet_Gen.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "VolumeGroups_PreBackup_MaximumSet",
+  "operationId": "VolumeGroups_Pre_Backup",
+  "parameters": {
+    "subscriptionId": "67B407F8-ED08-4144-B074-BA08FF4FB978",
+    "resourceGroupName": "rgelasticsan",
+    "elasticSanName": "elasticSanName",
+    "volumeGroupName": "volumeGroupName",
+    "parameters": {
+      "volumeNames": [
+        "volume1"
+      ]
+    },
+    "api-version": "2025-03-01"
```

Comment:
As I heard this feature is for preview, so it might should add to a preview API version.